cmd/promtool/main.go

```diff
@@ -148,11 +148,11 @@ func main() {
 	// TODO(aSquare14): add flag to set default block duration
 	importFilePath := openMetricsImportCmd.Arg("input file", "OpenMetrics file to read samples from.").Required().String()
 	importDBPath := openMetricsImportCmd.Arg("output directory", "Output directory for generated blocks.").Default(defaultDBPath).String()
-	importRulesCmd := importCmd.Command("rules", "Create new blocks of data from Prometheus data for new rules from recording rule files.")
+	importRulesCmd := importCmd.Command("rules", "Create blocks of past data for new recording rules.")
```

Comment:
```suggestion
	importRulesCmd := importCmd.Command("rules", "Create blocks of data for new recording rules.")
```

We do not enforce the data to be from the paste.